lib/errors/kuzzleError.js

```diff
@@ -7,13 +7,14 @@ const util = require('util');
  */
 class KuzzleError extends Error {
 
-  constructor(message, status, errorName = 'Undocumented error', code = 0) {
+  constructor(message, status, errorName = 'internal.errorsManager.undocumented_error', code = 0) {
```

Comment:
so... what's the point in concatenating these information in kuzzle if we have to split them afterward in the kuzzleError object?

Reply:
![image](https://user-images.githubusercontent.com/4447392/63361958-5b2e6a00-c371-11e9-9c45-da3c0cfb8ddd.png)